SPECS/libsolv/libsolv.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "libsolv-0.7.24.tar.gz": "62743265222a729c7fe94c40f7b90ccc1ac5568f5ee6df46884e7ce3c16c78c7"
+  "libsolv-0.7.28.tar.gz": "bd2406f498fea6086ae0eacbf8b188c98b380e59af2267170e6a7b7d715cb207"
```

Comment:
It seems `libsolv` was merged recently:
- #7719

Reply:
Updated this PR to no longer contain the libsolv upgrade.